shared/js/utils.js

```diff
@@ -29,10 +29,10 @@ require.scopes.utils = ( () => {
 
     function parseUserAgentString (uaString) {
         if (!uaString) uaString = window.navigator.userAgent
-        const rgx = uaString.match(/(Firefox|Chrome|Safari)\/([0-9]+)/)
+        const rgx = uaString.match(/(Firefox|Chrome|Safari)\/([0-9.]+)/)
         return {
             browser: rgx[1],
-            majorVersion: rgx[2]
+            version: rgx[2]
```

Comment:
I changed it to full version when I updated the feedback info, changing this for consistency.